### DISCUSSION TITLE
Does this software have nullcast support?

### DISCUSSION THREAD

**Discussion by fredknack** [plhery/node-twitter-api-v2#241] [Q&A]
This tool is working great for me. My only question is can it implement nullcast so that replies do not show up on the senders public timeline? I can't find any mention of it in the documentation.

If it is possible how is it implemented?

Thanks

**Comment by alkihis** [comment]
Hi,
nullcast is specific to Ads API right?
No endpoint wrapper exists for Ads API, but you can still use Ads API endpoints with this lib by using `.get`/`.post` methods.

**Reply by fredknack** [reply]
Thanks. I don't think it is specific to ads. For example if I have an app that responds to specific hashtags in a filtered stream nullcasting would make the apps response tweets not show up in the stream though they would still be public.
